_data/signed/QingGO.yaml

```diff
@@ -0,0 +1,2 @@
+name: 曾颖清 (QingGO)
```

Comment:
Hi there, I suppose your name in Chines fails CI\CD. I'm not trying to harass or smth. but currently everybody get this error
![image](https://user-images.githubusercontent.com/38350817/112617319-7dca1400-8e2d-11eb-849b-bbac8252cda5.png)

Reply:
In that case in my opinion the appropriate solution is to fix the CI script, not change the name.

Reply:
(Both yaml and Jekyll fully support Unicode.)

Reply:
I find that my PR did not pass the check. so i try to use PinYin to write my name in this PR, or maybe the check script should by modify.

Reply:
I think @Lesik is wright, there might be some problems with CI. Also I double-checked the verification script, it seems to work fine.

Reply:
glad to hear that